doc/articles/studio/Hot Design/hot-design-canvas.md

```diff
@@ -108,6 +108,47 @@ This opens the editor for the `UserControl`, allowing you to modify its internal
 
 <img src="Assets/canvas-edit-user-control.gif" height="600" alt="How to edit a UserControl from the canvas" />
 
+## Edit Flyouts from the Canvas
+
+You can edit flyouts in Hot Design in two main ways.
+
+### Using the Property Window
+
+You can locate the `Flyout` property in the **Property Window** and edit it using the **Complex Type Editor**. To learn more about how the Complex Type Editor works, refer to the [Complex Type Editor documentation](xref:Uno.HotDesign.Properties.Editors#complex-types).
+
+### Using the Flyout Editor
+
+The **Flyout Editor** allows you to edit the contents of the flyout directly. You can add, remove, or modify individual elements inside the flyout.
+To open the Flyout Editor, follow these steps:
+
+1. In the **Toolbar**, click the **More Options** button (three dots).
+2. Hover over the **Window** menu and uncheck the option **Show tools in-app**.
+   This will move the Toolbox, Elements window, and Property window to external windows, leaving only the Canvas in the main window.
+3. Enable **Interactive Mode** by clicking the **Play** button in the toolbar.
+4. Perform the action that opens the flyout (e.g., click the button that has a flyout).
+5. With the flyout still opened, disable **Interactive Mode** by clicking the **Pause** button in the Toolbar.
+   This will activate the **Flyout Editor**.
+
+Now you can freely make any changes to the flyout.
+
+<img src="Assets/canvas-edit-flyouts.gif" alt="How to edit flyouts from the canvas" />
+
+To exit the **Flyout Editor**, you can:
+
+- Click the **Enter Interactive Mode** button in the Toolbar (Play icon)
+- Or click the **Back** icon in the top-left corner of the **Canvas** or **Tree window**
+
+  ![alt text](Assets/canvas-leave-flyout-editor.png)
+
+  ![alt text](Assets/tree-leave-flyout-editor.png)
```

Comment:
Looks good just a very small nitpick here on adding the "alt text" to the images

Reply:
Good catch 😁